vibora/utils.py

```diff
@@ -131,9 +131,12 @@ def wait_server_available(host: str, port: int, timeout: int=10) -> None:
     :param port: TCP port used to connect.
     :return:
     """
-    sock = socket.socket()
-    sock.settimeout(timeout)
     while timeout > 0:
```

Comment:
The `sock`will be undefined, if arg `timeout`is less than 0 when func is calling.